### PR TITLE
VIVI-10087 get youtube thumbnail

### PR DESCRIPTION
**Pull request by mattd233**
We'd like to get the thumbnail image for video links for play content.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -95,14 +95,16 @@ module.exports.process = (output, origin) => {
   const subtitleFile = findBestSubtitleFile(subtitles) || findBestSubtitleFile(automatic_captions);
   const subtitleUrl = subtitleFile ? `${origin}/ytdl/vtt?suburi=${encodeURIComponent(subtitleFile.subs.url)}` : '';
   const title = data.title || '';
+  const thumbnail = data.thumbnail || '';
```
Comment (lucashasil) [inline index.js:98]: Is this just stored as a buffer or is this a URL to the actual thumbnail file?

Comment (mattd233) [reply index.js:98]: It's just the url, we actually need to proxy it through our backend for client to have access to it.

Comment (lucashasil) [reply index.js:98]: Yeah cool, was just making sure we aren't loading full image buffers in

**Review by lucashasil — APPROVED** [review]

**Review by sajithpvivi — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (mattd233) [reply index.js:98]: Even if we were, to optionally load it we have to change vivi-service-ytdl cuz the request is processed there.

**Pull request merged**